web/covid_web/server.py

```diff
@@ -69,10 +57,33 @@ def covid19_predictions_redirect():
         _run_flask_server(server=server, data_dir=data_dir)
 
 
-def _run_flask_server(server: Flask, data_dir: Path):
+def _setup_server(server: Flask, data_dir: Path):
+    @server.route("/")
+    def home():
+        return render_template("index.html")
+
+    @server.route("/covid19/normal/")
+    def covid19_normal_redirect():
+        return redirect(url_for("covid19_all_predictions"))
+
+    @server.route("/covid19/predictions/")
+    def covid19_predictions_redirect():
+        return redirect(url_for("covid19_single_predictions"))
+
     _create_prediction_apps(server=server, data_dir=data_dir)
     _create_simulation_apps(server=server, data_dir=data_dir)
     _create_rest(server=server, data_dir=data_dir)
+
+
+def uswgi_server(data_dir: Path) -> Flask:
```

Comment:
This function happens to be used by `uswgi`, but that doesn't mean that we need to call it that.

I think this file should be factored differently.
```suggestion
def setup_server(data_dir: Path) -> Flask:
server = Flask(__name__, template_folder=str(CURRENT_DIR))
... setting up all @server.route's ...
return server
```

That is, change `_setup_server` into a public function.

Reply:
Simplified.